Ginger/GingerCore/Platforms/PlatformsInfo/PlatformInfoBase.cs

```diff
@@ -75,6 +75,29 @@ public static PlatformInfoBase GetPlatformImpl(ePlatformType Platform)
                     return null;
             }
         }
+
+        public static PlatformInfoBase GetPlatformTargetApplication(string targetApp)
+        {
```

Comment:
what is that?????

Reply:
This method is used to get the Platform for the target application, this method will be required at many places where we have only target application and platform is required.